packages/lodestar/test/e2e/noEth1Sim.test.ts

```diff
@@ -0,0 +1,92 @@
+import {IBeaconParams} from "@chainsafe/lodestar-params";
+import {WinstonLogger} from "@chainsafe/lodestar-utils";
+import {getDevBeaconNode} from "../utils/node/beacon";
+import {waitForEvent} from "../utils/events/resolver";
+import {Checkpoint} from "@chainsafe/lodestar-types";
+import {getDevValidator} from "../utils/node/validator";
+import {Validator} from "@chainsafe/lodestar-validator/lib";
+import {BeaconNode} from "../../src/node";
+
+describe.only("no eth1 sim", function () {
+
+  const validatorsPerNode = 8;
+  const beaconParams: Partial<IBeaconParams> = {
+    SECONDS_PER_SLOT: 2,
+    SLOTS_PER_EPOCH: 8
+  };
+
+  let onDoneHandlers: (() => Promise<void>)[] = [];
+
+  for (const nodeCount of [1, 2]) {
```

Comment:
we already have test for single node :)

You should call multinode test or something